src/Content/ContentReaders/SoundEffectReader.cs

```diff
@@ -53,10 +53,38 @@ SoundEffect existingInstance
 			uint nAvgBytesPerSec = Swap(se, input.ReadUInt32());
 			ushort nBlockAlign = Swap(se, input.ReadUInt16());
 			ushort wBitsPerSample = Swap(se, input.ReadUInt16());
-			/* ushort cbSize =*/ input.ReadUInt16();
-
-			// Seek past the rest of this crap (cannot seek though!)
-			input.ReadBytes((int) (formatLength - 18));
+			ushort cbSize = Swap(se, input.ReadUInt16());
+
+			byte[] extra = null;
```

Comment:
Do we just want to use a byte* and stackalloc to prevent GC activity?

Reply:
Would be nice, but it'd require memcpy in the SoundEffect constructor to copy from extraData to formatPtr + offs. Buffer.BlockCopy doesn't exist in old .NET runtimes and SDL_memcpy exists but is off limits as this is outside of SDL2_FNAPlatform.

Thinking about it a bit more after talking on it on Discord: maaaaybe only for FNA.Core / a compile time flag? Might not be worth the effort tho 😅